MediaGalleryUi/view/adminhtml/web/js/image-uploader.js

```diff
@@ -92,6 +93,16 @@ define([
 
                         this.count() < 2 || this.mediaGridMessages().scheduleCleanup();
 
+                        return;
+                    } else if (!this.isFileNameLengthExceeded(data.files[0]).passed) {
+                        this.mediaGridMessages().add(
+                           'error',
+                           $.mage.__('Cannot upload <b>' + data.files[0].name +
+                                     '</b>. Filename is too long, must be 90 chracters or less.')
+                        );
+
+                        this.count() < 2 || this.mediaGridMessages().scheduleCleanup();
```

Comment:
It may worth extracting this into a function, I've noticed we are calling the same thing on the if condition abode this one, what do you think?

Reply:
yes i'm also was thinking about this, will move this to the separate method :+1: